src/drivers/config/drivers.md

```diff
@@ -21,18 +21,24 @@ drivers:
     set-nvidia-runtme: true
 ```
 
+Or if your cluster has already installed nvidia-driver, and do not need pai to install it
+again, then you can provide this info in your service-configuration.yaml like:
+
+```yaml
+drivers:
```

Comment:
hi @xudifsd  does this pr cherry pick to pai/aks ? i think is useful

Reply:
why? you need to change the config?

Reply:
if not change could work on AKS, no need is ok

Reply:
yes, the default value /usr/local/driver is where aks's driver exist, so no need to cherry-pick to aks.

Reply:
ok thanks